platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -780,38 +546,61 @@ public class SwirldsPlatform implements Platform, ConnectionTracker, Startable {
             /* validates events received from gossip */
             final EventValidator eventValidator = new EventValidator(eventValidators, eventIntake::addUnlinkedEvent);
 
-            final EventTaskDispatcher taskDispatcher = new EventTaskDispatcher(
+            taskDispatcher = new EventTaskDispatcher(
                     time,
                     eventValidator,
                     eventCreator,
                     eventIntake::addUnlinkedEvent,
                     eventIntakeMetrics,
                     intakeCycleStats);
 
-            final InterruptableConsumer<EventIntakeTask> intakeHandler;
-            if (chatterConfig.useChatter()) {
-                intakeCycle = new SequenceCycle<>(taskDispatcher::dispatchTask);
-                intakeHandler = intakeCycle;
-            } else {
-                intakeCycle = null;
-                intakeHandler = taskDispatcher::dispatchTask;
-            }
-
             intakeQueue = components.add(new QueueThreadConfiguration<EventIntakeTask>(threadManager)
                     .setNodeId(selfId.id())
                     .setComponent(PLATFORM_THREAD_POOL_NAME)
                     .setThreadName("event-intake")
-                    .setHandler(intakeHandler)
+                    .setHandler(e -> getGossip().getEventIntakeLambda().accept(e))
```

Comment:
cant it be just `getGossip().getEventIntakeLambda()`?

Reply:
Unfortunately not. This is required to resolve a circular dependency. Gossip requires the intqke queue, and the intake queue requires gossip. So we can't call `getGossip()` until construction has completely finished.

Reply:
Added a comment explaining:

```
                    // There is a circular dependency between the intake queue and gossip,
                    // which the handler lambda sidesteps (since the lambda is not invoked
                    // until after all things have been constructed).
                    .setHandler(e -> getGossip().getEventIntakeLambda().accept(e))
```